org.gnome.Evince.json

```diff
@@ -121,9 +138,15 @@
             ],
             "sources": [
                 {
-                    "type": "archive",
-                    "url": "https://download.gnome.org/sources/libgxps/0.3/libgxps-0.3.1.tar.xz",
-                    "sha256": "1a939fc8fcea9471b7eca46b1ac90cff89a30d26f65c7c9a375a4bf91223fa94"
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/libgxps.git",
+                    "tag": "0.3.1",
+                    "commit": "1e508695c38ede224464ab3117777a2e65cf17ae",
```

Comment:
why do we need to use the git version and not the released one? I prefer the former.